Dockerfile

```diff
@@ -14,10 +14,14 @@ RUN apt-get update && apt-get install -y \
     libssl-dev \
     wget \
     zip && \
-    # install go 1.7
-    add-apt-repository -y ppa:longsleep/golang-backports && \
-    apt-get update && apt-get install -y golang-go && \
     rm -rf /var/lib/apt/lists/*
+
+# Install go 1.8.5
+RUN curl -O https://storage.googleapis.com/golang/go1.8.5.linux-amd64.tar.gz
```

Comment:
This image gets pulled *a lot* so it would definitely be a small, but meaningful performance increase to tie these three `RUN` commands together before shipping to reduce extra layers

Reply:
Done